kratos/utilities/result_dabatase.h

```diff
@@ -0,0 +1,542 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+
+#if !defined(KRATOS_RESULT_DATABASE_H_INCLUDED )
+#define  KRATOS_RESULT_DATABASE_H_INCLUDED
+
+// System includes
+
+// External includes
+#include <unordered_map>
+
+// Project includes
+#include "includes/table.h"
+
+namespace Kratos
+{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/**
+ * @class EntityDatabase
+ * @ingroup KratosCore
+ * @brief This class stores the results of a entity
+ * @author Vicente Mataix Ferrandiz
+*/
+class EntityDatabase
+    : public std::vector<std::vector<Table<double, double>*>>
```

Comment:
this

Reply:
But, what do you suggest?

Reply:
no idea
discussing if this is a good idea or not...?

did you google before doing this? E.g. from what I read the destructor of `std::vector` is not `virtual`

Reply:
Agree with @philbucher, if you need a container as the basis for the class you should follow the design of other classes in Kratos that do the same. 

Almost everything in `kratos/containers`. For example https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/containers/global_pointers_vector.h#L215-L319